### ISSUE TITLE
Excellent use of Mocha beforeEach and afterEach hook

### ISSUE THREAD

**Issue by h3xar0n** [1334/vidbits#3]
You did a great job cleaning up with these hooks. Out of curiosity, did you find this by reading Mocha's docs, or see it elsewhere?

https://github.com/1334/vidbits/blob/b1f93573d14e43fecd617d94ac2f1973e2b6332a/test/models/video-test.js#L5-L7

**Comment by 1334** [comment]
yeah, if I remember correctly one of the previous exercises of the course had a similar solution, but I definitely skimmed over the documentation for a better understanding of the framework
